core/domain/wipeout_service.py

```diff
@@ -466,6 +489,29 @@ def delete_user(
     _delete_models(user_id, models.Names.LEARNER_GROUP)
 
 
+def _verify_profile_picture_is_deleted(username: str) -> bool:
+    """Verify that the profile picture is deleted.
+
+    Args:
+        username: str. The username of the user.
+
+    Returns:
+        bool. True when the profile picture is deleted else False.
+    """
+    if username is None:
+        return True
+    fs = fs_services.GcsFileSystem(feconf.ENTITY_TYPE_USER, username)
+    filename_png = 'profile_picture.png'
+    filename_webp = 'profile_picture.webp'
+    if fs.isfile(filename_png) or fs.isfile(filename_webp):
+        logging.error(
+            '%s Profile picture is not deleted of user having '
```

Comment:
of --> for

Also specify whether it's the png, webp, or "png and webp".

Reply:
Done